### PR TITLE
New homepage

### PR DESCRIPTION
**Pull request by scpaye506**

### CODE REVIEW

File: handlers/Home/components/Services/index.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  constructor(props: mixed, context: mixed): void {
+    super(props, context);
+
+    this.state = {target: null};
+    this.handleSelectSegment = this.handleSelectSegment.bind(this);
+  }
+
+  handleSelectSegment(event: mixed): void {
+    if (event) event.preventDefault();
+    var target = event.target.closest('a').getAttribute('href').replace('#', '');
```
Comment (iamdustan) [inline handlers/Home/components/Services/index.js]: https://developer.mozilla.org/en-US/docs/Web/API/Element/closest

this is a pretty sketchy API that will likely not work for most of our users. Is there a more declarative way we can handle it that doesn’t require DOM traversal?

File: handlers/Home/components/Services/index.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    this.setState({target});
+  }
+
+  render(): ReactElement {
+    return (
+      <div className="HomeServices">
+        <ul className="HomeServices-list">
+          {SERVICES.map(s => (
+            <li ref={s[1]} style={{backgroundImage: `url(${s[2]})`}}className={`HomeServices-item ${s[1] === this.state.target ? 'is-active' : ''}`} key={s[0]}>
```
Comment (iamdustan) [inline handlers/Home/components/Services/index.js]: this is pretty dense. Could you newline each attribute?

File: handlers/Home/components/Services/styles.css [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+  display: inline-block;
+  float: left;
+  height: 25vw;
+  margin-bottom: 0px;
+  position: relative;
+  width: 33.3333%;
+  background-size: cover;
+}
+
+.HomeServices-item:after, .HomeServices-item:before {
```
Comment (iamdustan) [inline handlers/Home/components/Services/styles.css]: I swear I’m the only person in all of Skookum that refuses to do single-line comma selectors (here and a few down)

Comment (iamdustan) [inline handlers/Home/components/Services/styles.css]: :)

File: handlers/Home/components/Services/styles.css [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+  opacity: 0;
+  font-size: 1.1em;
+  font-weight: 300;
+  line-height: 1.4em;
+  transform: translate3d(0, 0, 0);
+}
+
+.HomeServices-item.is-active .HomeServices-content {
+  background-color: rgba(255,255,255,0.92);
+  box-shadow: 0px 2px 15px 0px rgba(0,0,0,0.23);
```
Comment (iamdustan) [inline handlers/Home/components/Services/styles.css]: Mr. OCD in me want to enforce `, ` white spacing and alphabetical properties for all of our CSS declarations :innocent:

File: handlers/Home/components/Services/styles.css [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+.HomeServices-item.is-active  .HomeServices-title {
+  /*padding-top: 12%;*/
+  color: #FAAB18;
+  transition: all ease-out 150ms;
+  transition-delay: 100ms;
+  transform: translate3d(0, -48%, 0);
+
+}
+
+.HomeServices-item.is-active  .HomeServices-description {
```
Comment (iamdustan) [inline handlers/Home/components/Services/styles.css]: I’m noticing quite a few nested selectors. Mind expounding on the reasoning? I assume this is breaking the suit linter at current.

File: handlers/Home/index.js [hunk 6]
```diff
@@ -17,27 +18,35 @@ class Home extends React.Component {
...
       <div className="Home">
         <Hero
-          title={<img src="/public/images/wordmark.svg" alt="Skookum" />}
+          childrenPosition="before"
           color="black"
-          image="/public/images/hero-case-studies.png" />
+          image="/public/images/hero-case-studies.png">
+          <div className="Home-banner">
+            <div className="Home-bannerTitle">
+              Custom software solutions for companies and the <span style={{color: '#FAAB18'}}>people</span> they serve.
```
Comment (iamdustan) [inline handlers/Home/index.js]: Could we give this hex value a name?

File: handlers/Home/components/Services/index.js (hunk 1 above)
Comment (scpaye506) [inline handlers/Home/components/Services/index.js]: all of this is removed in the next PR

File: handlers/Home/components/Services/styles.css (hunk 5 above)
Comment (scpaye506) [inline handlers/Home/components/Services/styles.css]: Interestingly, it doesn't break it. This is changes a bit in the second PR though ```.is-active``` changes to ```:hover```, essentially what's happening is parent is being hovered and each nested div is being styled slightly different

**Pull request merged**